docs/requirements.txt

```diff
@@ -1,4 +1,4 @@
-sphinx==3.5.4
+sphinx==4.3.2
```

Comment:
Sphinx is already on their 4.4 release. Maybe we shouldn't pin this at all?

Reply:
Pinning is good, otherwise updates might break build pipeline